dashboard/app/views/levels/editors/fields/_authored_hints.haml

```diff
@@ -67,6 +67,15 @@
             'markdown',
             {
               callback: function(editor, change) {
+                //Looking for `.markdown_preview div --> div --> [XML Blockly divs]` structure
+                //gets .markdown_preview div
+                let parent = space.find('.markdown_preview').get(0); 
+                //When XML is inserted, it creates a smaller div
+                let subParent = parent.querySelector("div");
```

Comment:
Looking [here](https://github.com/code-dot-org/code-dot-org/blob/bc7ec7398e27a7bff334222ebb26613a9a086358/apps/src/templates/instructions/utils.js#L132-L137), it appears that it will be a `span` if the XML had a `<p>` for its parent.  Is that a possible scenario?

Reply:
Good find, Brendan, and yes it's possible. Testing an "inline" block on Dan's branch reveals the bug is still present because the workspace is in a span element:

https://user-images.githubusercontent.com/43474485/236836879-ca3e21cb-c26c-499d-a755-428c7d59ff04.mp4